lib/trento/domain/cluster/cluster.ex

```diff
@@ -163,14 +194,17 @@ defmodule Trento.Domain.Cluster do
     maybe_emit_host_added_to_cluster_event(cluster, host_id)
   end
 
+  # When a message arrives from a non DC host who belongs to a cluster, we add that host to the cluster
```

Comment:
This comment makes little sense, as it talks about a `non DC` but the code is explicitily for a DC host